crates/dc_bundle/src/definition/element.rs

```diff
@@ -490,3 +490,9 @@ impl FontWeight {
     /// Black weight (900), the thickest value.
     pub const BLACK: FontWeight = FontWeight::from_num(900.0);
 }
+
+impl Vector {
+    pub fn is_valid(&self) -> bool {
```

Comment:
`is_non_zero`? `has_magnitude`? Something like that, bc a zero vector is "valid" in many contexts.

Reply:
Actually I don't think it was being used anywhere so I just deleted it.